prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1089,15 +1081,33 @@ int rxr_ep_init(struct rxr_ep *ep)
 
 		if (ret)
 			goto err_free;
+
+		ret = ofi_bufpool_grow(ep->rx_unexp_pkt_pool);
```

Comment:
Doxygen comment for rxr_ep_init() needs to be updated to reflect this change.

Reply:
Good catch! fixed in new revision.